sds_data_manager/stacks/sds_data_manager_stack.py

```diff
@@ -0,0 +1,216 @@
+# Standard
+import pathlib
+
+# Installed
+import aws_cdk as cdk
+from constructs import Construct
+from aws_cdk import (
+    Stack,
+    Environment,
+    RemovalPolicy,
+    aws_iam as iam,
+    aws_s3 as s3,
+    aws_s3_deployment as s3_deploy,
+    aws_secretsmanager as secrets,
+    aws_lambda as lambda_,
+    aws_lambda_event_sources,
+    aws_lambda_python_alpha as lambda_alpha_
+)
+
+# Local
+from .opensearch_stack import OpenSearch
+
+
+class SdsDataManager(Stack):
+    """Stack for Data Management."""
+    def __init__(self, scope: Construct,
+                 construct_id: str,
+                 sds_id: str,
+                 opensearch: OpenSearch,
+                 env: Environment,
+                 **kwargs) -> None:
+        """SdsDataManagerStack
+
+        Parameters
+        ----------
+        scope : App
+        construct_id : str
+        sds_id: str
+        env : Environment
```

Comment:
You're missing opensearch in the doc strings. In general, it makes sense that this class depends on opensearch, but I'd like to see a note about that here, since I know those dependencies can be tricky. Just something like "this class depends on opensearch, which is built with opensearch_stack.py" would make it clear that there is another class needed before you call this one. I don't expect this stack to get used much outside of stackbuilder.py but similar things have really tripped me up in other projects, lol